dorado/routines.py

```diff
@@ -949,3 +949,191 @@ def snake_plots(grid,
                     +os.sep+'snakefig'+str(ii)+'.png',
                     bbox_inches='tight')
         plt.close()
+
+
+def show_nourishment_area(visit_freq, grid=None, walk_data=None,
+                          cmap='Reds', sigma=0.7, min_alpha=0,
+                          show_seed=True, seed_color='dodgerblue'):
+    """Plot a smoothed, normalized heatmap of particle visit frequency
+
+    Function will plot the history of particle travel locations in walk_data
+    as a heatmap overtop the specified grid, using the output of
+    particle_track.nourishment_area(). Colors indicate number of instances
+    in which that cell was occupied by a particle.
+
+    **Inputs** :
+
+        visit_freq : `numpy.ndarray`
+            A 2-D grid of normalized particle visit frequencies, i.e. the
+            output of the dorado.particle_track.nourishment_area() function
+
+        grid : `numpy.ndarray`, optional
+            An optional 2-D grid upon which the particles will be plotted.
+            Examples of grids that might be nice to use are
+            `dorado.particle_track.modelParams.depth`,
+            `dorado.particle_track.modelParams.stage`,
+            `dorado.particle_track.modelParams.topography`.
+
+        walk_data : `dict`, optional
+            The dictionary with the particle information, which is used to
+            show the seed location. This is the output from one of the other
+            routines or the :obj:dorado.particle_track.run_iteration() function.
```

Comment:
For this to create a proper link, the object reference needs to be :obj:'dorado.particle_track.Particles.run_iteration()' where instead of the single apostrophe, the backwards one (`) is used.

Same on line 1037. We have this issue in the `plot_state` docstring around like 789, so would be great to make that minor fix as part of this PR as well.